xenoclassify.wdl

```diff
@@ -0,0 +1,137 @@
+version 1.0
+
+workflow xenoClassify {
+input {
+        File fastqR1
+	File fastqR2
```

Comment:
`File fastqR2` -> `File+ fastqR2` to support single end

Reply:
implemented